src_assets/common/assets/web/config.html

```diff
@@ -391,19 +391,19 @@ <h2 class="accordion-header">
             <pre>tools\dxgi-info.exe</pre>
           </div>
         </div>
-        <div class="mb-3" v-if="platform === 'linux'">
-          <label for="output_name" class="form-label">{{ $t('config.output_name_linux') }}</label>
+        <div class="mb-3" v-if="platform === 'linux' || platform === 'macos'">
+          <label for="output_name" class="form-label">{{ $t('config.output_name_unix') }}</label>
           <input type="text" class="form-control" id="output_name" placeholder="0" v-model="config.output_name" />
           <div class="form-text">
-            {{ $t('config.output_name_desc_linux') }}<br>
+            {{ $t('config.output_name_desc_unix') }}<br>
             <br>
             <pre style="white-space: pre-line;">
-              Info: Detecting connected monitors
-              Info: Detected monitor 0: DVI-D-0, connected: false
-              Info: Detected monitor 1: HDMI-0, connected: true
-              Info: Detected monitor 2: DP-0, connected: true
-              Info: Detected monitor 3: DP-1, connected: false
-              Info: Detected monitor 4: DVI-D-1, connected: false
+              Info: Detecting displays...
+              Info: Detected display: DVI-D-0 (id: 0) connected: false
+              Info: Detected display: HDMI-0 (id: 1) connected: true
+              Info: Detected display: DP-0 (id: 2) connected: true
+              Info: Detected display: DP-1 (id: 3) connected: false
+              Info: Detected display: DVI-D-1 (id: 4) connected: false
```

Comment:
Do you think there should be a macOS example and a Linux example? We can use `v-if="platform === 'linux'` if so... And if we do that option the docs should probably have both examples as well.

Otherwise, do you think this sample should match what is in the docs?

I'm fine with either option as long as it's consistent.

Reply:
In Linux, sunshine currently list interface names rather than monitor's product name, which differs from macOS. Moreover, the display ID counting mechanism differs between the two platforms. I believe it would be more appropriate to separate examples for macOS and Linux in both documentation and config ui page for clarity.

Reply:
While I am familiar with some aspects of Linux, to maintain consistent log formats across two platforms, it might be desirable for Linux to also retrieve monitor product names like macOS does for displays. However, the feasibility and ease of implementation would depend on the existing APIs or libraries available in Linux. It might require additional effort to adapt those functions to return product names instead of interface names.